pkg/logs/sender/destinations_context.go

```diff
@@ -0,0 +1,51 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package sender
+
+import (
+	"context"
+	"sync"
+)
+
+// A DestinationsContext manages senders and allows us to "unclog" the pipeline
+// when trying to stop it and failing to send messages.
+type DestinationsContext struct {
+	context context.Context
+	cancel  context.CancelFunc
+	mutex   sync.Mutex
+}
+
+// NewDestinationsContext returns an initialized ConnectionManager
+func NewDestinationsContext() *DestinationsContext {
+	return &DestinationsContext{}
+}
+
+// Start creates a context that will be cancelled on Stop()
+func (sm *DestinationsContext) Start() {
```

Comment:
what does `sm` stand for ? can you change this to `dc` please ?

Reply:
it stands for "bad copy paste"